### PR TITLE
#3 Build out REST Endpoints for Resources (POST /api/v1/auth/create-user) 

### PR DESCRIPTION
**Pull request by LamidoKing**
Description
Having POST on /api/v1auth//create-user

How Has This Been Tested?
POST /api/v1/auth/create-user

curl -X POST localhost:3000 /api/v1auth/create-user -H 'content-Type: application/json' \ -d '{"email": "lamido12@gmail.com", "name": "lamido", "password": "Mypassword123", "passwordConfirmation": "Mypassword123"}'

#3

### CODE REVIEW

File: __tests__/genaralTestConfi.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return connect;
+};
+
+export const mongoseDisconnect = async () => {
+  const disconnect = await mongoose.connection.close();
+  return disconnect;
+};
+
+export const userData = {
+  email: 'team04@buidsdg.com',
```
Comment (komputarist) [inline __tests__/genaralTestConfi.js:20]: I am sure this email is dummy, but shouldn't it read 'team042@buildsdg.com'

**Review by komputarist — APPROVED** [review]
Looks good

**Review by matbami — COMMENTED** [review]
Nice one

**Review by matbami — APPROVED** [review]

File: __tests__/genaralTestConfi.js (hunk 1 above)
Comment (LamidoKing) [reply __tests__/genaralTestConfi.js:20]: will be fix in next PR

**Review by tochiOz — APPROVED** [review]

**Pull request merged**